styles/base.less

```diff
@@ -466,3 +466,16 @@ atom-text-editor.is-focused .line.cursor-line {
   font-style: italic;
   color: #ffc600;
 }
+
+/*Nested elements in JSX*/
+.JSXNested {
+  color: #F8F8F8;
+}
+
+.entity.function.js {
+  color: #CCCCCC;
+}
+
+.entity.function.method {
+  color: #FFC600;
+}
```

Comment:
I assume this is here to override the `.entity.function.js` declaration in the method context? Given that I think that declaration should go away, I think this one should go away as well, since it's already handled in https://github.com/wesbos/cobalt2-atom/blob/v0.5.0/styles/base.less#L88-L90